recipes/linb-any/all/conanfile.py

```diff
@@ -0,0 +1,37 @@
+from conans import ConanFile, tools
+
+required_conan_version = ">=1.33.0"
+
+class LinbAnyConan(ConanFile):
+    name = "linb-any"
```

Comment:
I'm curious about this package name. Why linb ?

Reply:
linb is name of namespace in any.hpp.
Should I use "thelink2012" which is the author account?

Reply:
Yes, please. When doing a quick search, I can't find any result for `linb-any` on Google/Github, but I can find for `thelink2012-any`.